web/src/stories/FrozenTableStory.js

```diff
@@ -0,0 +1,157 @@
+                  <td className="mono right-align nowrap" headers="quarterly-budget-summary-hitAndHie-fy-2020 quarterly-budget-summary-hitAndHie-fy-2020-q1">--</td>
+                  <td className="mono right-align nowrap" headers="quarterly-budget-summary-hitAndHie-fy-2020 quarterly-budget-summary-hitAndHie-fy-2020-q2">--</td>
+                  <td className="mono right-align nowrap" headers="quarterly-budget-summary-hitAndHie-fy-2020 quarterly-budget-summary-hitAndHie-fy-2020-q3">--</td>
+                  <td className="mono right-align nowrap" headers="quarterly-budget-summary-hitAndHie-fy-2020 quarterly-budget-summary-hitAndHie-fy-2020-q4">--</td>
+                  <td className="bold mono right-align nowrap" headers="quarterly-budget-summary-hitAndHie-fy-2020 quarterly-budget-summary-hitAndHie-fy-2020-subtotal">--</td>
+                  <td className="bold mono right-align nowrap" headers="quarterly-budget-summary-hitAndHie-total2 quarterly-budget-summary-hitAndHie-total">--</td>
+                </tr>
+              </tbody>
+            </table>
+          </div>
+        </div>
+      </RenderViewbox>
+      <h2>The basic structure to create this is as follows:</h2>
+      <pre>
+        <code>
+          <div className="table-frozen-wrapper">
+            <div className="table-frozen-scroller">
+              <table className="table-cms table-frozen-left-pane" aria-hidden="true">
+                <thead>
+                  <tr>
+                    <th className="table-frozen-null-cell">--</th>
+                  </tr>
+                  <tr>
+                    <th className="table-frozen-null-cell">--</th>
+                  </tr>
+                </thead>
+                <tbody>
+                  <tr>
+                    <td>In-House Costs</td>
+                    <td className="mono right-align nowrap"></td>
+                  </tr>
+                  <tr>
+                    <td>Private Contractor Costs</td>
+                    <td className="mono right-align nowrap"></td>
+                  </tr>
+                  <tr className="bold">
+                    <td>Total Enhanced FFP</td>
+                    <td className="mono right-align nowrap"></td>
+                  </tr>
+                </tbody>
+              </table>
+              <table className="table-cms table-frozen-data">
+                [your table header and body go here]
+              </table>
+            </div>
+          </div>
+        </code>
+      </pre>
+      <p>
+        Essentially, this renders a second table on top of the first one. The second table is what creates the scrollable sticky pane. In order to get the height of the columns in the second table right, we render the first column of content, also (this is especially helpful when the table contains input elements).
```

Comment:
This one is a little confusing because "second table" refers to the first one, and "first one" refers to the second table.  How about something like this?

```suggestion
        Essentially, this renders a table of just headers on top of the full table.  The first table is what creates the scrollable sticky pane.  In order to get the height of the columns in the header table right, we render the first column of real content, also (this is especially helpful when the table contains input elements).
```

(This was supposed to be in my review, but I guess I never pushed the button...)